core/src/main/kotlin/fr/sncf/osrd/api/api_v2/standalone_sim/SimulationResponse.kt

```diff
@@ -19,8 +20,9 @@ class SimulationSuccess(
     val base: ReportTrain,
     val provisional: ReportTrain,
     @Json(name = "final_output") val finalOutput: CompleteReportTrain,
-    val mrsp: MRSPResponse,
-    @Json(name = "electrical_profiles") val electricalProfiles: RangeValues<ElectricalProfileValue>,
+    val mrsp: BlockPathRangeValues<SpeedLimitProperty>,
+    @Json(name = "electrical_profiles")
+    val electricalProfiles: BlockPathRangeValues<ElectricalProfileValue>,
```

Comment:
Using "BlockPath" ranges here as it relates to the (Block)`Path`, not the `TravelledPath` if I'm correct.

Reply:
That feels wrong, the "block path" (the one where offsets always start at the border of a block) should only be an internal core representation. We've had confusions with `Path` / `TravelledPath` in the schemas (as the name `Offset<Path>` feels closer to what we want to describe), but that confusion should only be a naming thing. I don't think any actual `Offset<Path>` should be returned.

Reply:
As discussed, I was mistaken on that, I tried to document/rename a bit around that: https://github.com/OpenRailAssociation/osrd/pull/8351/commits/b4c6f6f013b673a29e906c167d7bc2d24f5f92a0#diff-4c2dee00411e3f70730c838ef56323cdd8c1272c53725ec2db95cabc701e134eR23
This is part of a greater effort to be made step-by-step 🙂